src/commonmark/en/content/user/using-the-maps-app.md

```diff
@@ -715,58 +709,67 @@ configuration dialog.
 
     ![](resources/images/maps/maps_tracked_entity_layer_dialog_DATA.png)
 
-      - Select the **Tracked Entity Type** you want to show on the map.
+    - Select the **Tracked Entity Type** you want to show on the map.
+
+    - Select a **Program** where the tracked entities belong.
+
+    - Set the **Program status** to be **Active** or **Completed**.
+
+    - Set the **Follow up** status of the tracked entity for the given
+      program.
+
+2.  In the **Relationships** tab
+
+    ![](resources/images/maps/maps_tracked_entity_layer_dialog_RELATIONSHIPS.png)
 
-      - Select a **Program** where the tracked entities belong.
+    _**NOTE:** Displaying tracked entity relationships in Maps is an experimental feature_
```

Comment:
Notes (asides) should be written as block quotes in a particular form (see line 407)